src/usr/share/netcheck/netcheck.py

```diff
@@ -176,7 +176,7 @@ def _check_connection_and_check_dns(self, network_name):
                           'Internet over %s.' % network_name)
 
         overall_success = False
-        connection_active = self.network_meta.is_connected(network_name)
+        connection_active = self.network_helper.network_is_ready(network_name)
```

Comment:
While I'm not sold on the new method name (I'll save judgment on that until later), if we keep this method name, the assigned variable name should probably also change.

Reply:
After reviewing the rest of the code, I'd probably call this is_activated or network_is_activated.